game.rb

```diff
@@ -0,0 +1,16 @@
+require './item'
+
+class Game < Item
+  attr_accessor :platform, :last_played_at
+
+  def initialize(name, platform, last_played_at)
+    super(name)
+    @platform = platform
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    parent = super
```

Comment:
In the can_be_archived? method, when calling super, you should use super() without any arguments since the Item class doesn't have a can_be_archived? method.